keywords/SyncGateway.py

```diff
@@ -273,6 +273,8 @@ def start_sync_gateways(self, cluster_config, url=None, config=None):
             self.server_port = 18091
             self.server_scheme = "https"
 
+        if is_ipv6:
```

Comment:
same as above

Reply:
wrong implementation, but it would be true for IPv6 and IPv4 too. Fixing it.